### PR TITLE
Initial contribution guide line

### PR DESCRIPTION
**Pull request by redcatbear**

### CODE REVIEW

**Comment by redcatbear** [comment]
@chku2469: added your ideas? Is this what you had in mind?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Thank you for your interest in helping us to make OpenFastTrace (OFT) better!
+
+# Goal of This Document
+This document aims at answering all the questions potential contributers to OFT might have before feeling ready to get startet.
```
Comment (kaklakariada) [inline CONTRIBUTING.md]: startet -> started

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# What Kind of Contributions can I Make?
+
+## Contributing Code
+If you are a programmer and want to help us improve the implementation, test cases or design of OFT please create a branch of the current `develop` branch of OFT using [git](https://git-scm.com/) and make your changes on that branch.
+
+Then please create a pull request and ask for a review by one of the core team members (e.g. `redcatbear` or `kaklakariada`).
+The reviewers will either ask you to work in review findings or if there are none, merge your branch.
+
+For code contributions we kindly ask you to either pick or create a GitHub [issue in the OFT project](https://github.com/itsallcode/openfasttrace/issues) first which serves context in which the code commit lives.
```
Comment (kaklakariada) [inline CONTRIBUTING.md]: Is it necessary to create an issue first? We should make it as simple as possible for contributors.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+If you are a programmer, a code contribution in form of an automatic unit test case would be most appreciated, since this will make reproduction of the issue easier and prevent future regressions.
+
+## Contributing to the User Guide
+Maybe you are good at explaining how to use OFT to end users? Help us improve the [user guide](doc/user_guide.md)!
+
+## Translations
+We plan to make OFT multilingual. If you want to provide a translation, feel free to contact us. Messages in OFT and the user guide are prime candidates for translation.
+
+## Fields of Application
+Found a clever way to apply OFT in a new field of application? Share your story with the OFT community!
```
Comment (kaklakariada) [inline CONTRIBUTING.md]: How to share? Maybe add to the user guide?

File: CONTRIBUTING.md (hunk 1 above)
Comment (redcatbear) [reply CONTRIBUTING.md]: Fixed.

**Pull request merged**